Climate_police/tests/hw7-mao/pollution_map.py

```diff
@@ -0,0 +1,44 @@
+import plotly.offline as py
+py.init_notebook_mode()
```

Comment:
You are right. I intended to call this function in the notebook. We have to decide the way we present our results.